tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -152,3 +158,30 @@ def test_arxiv_derive_inspire_categories_does_nothing_with_existing_categories()
 
     assert validate(result, inspire_categories_schema) is None
     assert expected == result
+
+
+@patch('inspirehep.modules.workflows.tasks.arxiv.os')
+def test_arxiv_author_list_handles_auto_ignore_from_arxiv(mock_os):
+    data = {
+        'arxiv_eprints': {
+            'value': ['1703.09986']
+        }
+    }
+    mock_arxiv_tarball = MockFiles({
+        '1703.09986.tar.gz': AttrDict({
+            'file': AttrDict({
+                'uri': resource_filename(__name__, 'fixtures/1703.09986.tar.gz')
+            })
+        })
+    })
+    mock_obj = MockObj(data, None, mock_arxiv_tarball)
+    mock_eng = MockEng()
+
+    try:
+        temporary_dir = mkdtemp()
+        mock_os.path.abspath.return_value = mkdtemp()
```

Comment:
I guess the second `mkdtemp` call is a typo? You probably wanted to use the `temporary_dir` defined above...